### DISCUSSION TITLE
Could you please add Tuya Light and Human Presence Detector 5.8GHz to the supported devices?

### DISCUSSION THREAD

**Discussion by iqambassador** [zigpy/zha-device-handlers#1845] [Q&A]
Please add the following device to ZHA library.
Model ZC-QY manufactured by Tuya
Zigbee ID: TS0601 | _TZE200_jva8ink8
https://zigbee.blakadder.com/Tuya_ZC-QY.html

**Comment by Hedda** [comment]
FYI, there already looks to be an existing device support request (e.i. feature request) for TS0601 | _TZE200_jva8ink8 under issues here:

https://github.com/zigpy/zha-device-handlers/issues/1781

Note feature requests should be posted under issues instead of under discussions, e.i. submit "Device support request" in the issues tab:

https://github.com/zigpy/zha-device-handlers/issues/new/choose

The future tip is to always look and search for existing requests 

https://github.com/zigpy/zha-device-handlers/issues

Better example for your device; [Device Support Request] Tuya Light and Human Presence Detector 5.8GHz (TS0601 - _TZE200_jva8ink8)

Also be sure to help fill out the template/form with Zigbee Device Signature and debug logs as requested. 

New issue -> Device support request -> Get started

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
A clear and concise description of what you want to happen.

<details>

<summary>Device signature</summary>

```yaml
Paste the device signature here.
Don't remove the extra line breaks outside the ``` marks.
```

</details>

<details>

<summary>Diagnostic information</summary>

```yaml
Paste the diagnostic information here.
Don't remove the extra line breaks outside the ``` marks.
```

</details>

<details>
<summary>Additional logs</summary>

```
Paste any additional debug logs here.
Don't remove the extra line breaks outside the ``` marks.
```

</details>

**Additional context**
Add any other context or screenshots about the feature request here.
